swarm/Cargo.toml

```diff
@@ -31,6 +31,7 @@ multistream-select = { workspace = true }
 [target.'cfg(not(any(target_os = "emscripten", target_os = "wasi", target_os = "unknown")))'.dependencies]
 async-std = { version = "1.6.2", optional = true }
 tokio = { version = "1.29", features = ["rt"], optional = true }
+tracing = "0.1.37"
```

Comment:
I am assuming that `tracing` does have WASM support and that it should really go into the `[dependencies]` section. Is that correct?